aggregateur/validators.py

```diff
@@ -40,7 +46,24 @@ def extract(self):
         raise NotImplementedError
 
     def metadata(self):
-        raise NotImplementedError
+        slug = self.repo.slug
+
+        return {
+            "slug": slug,
+            "title": self.title,
+            "description": self.description,
+            "homepage": self.homepage,
+            "type": self.repo.schema_type,
+            "consolidation_tags": self.consolidation_tags(slug),
+            "email": self.repo.email,
+            "version": self.repo.current_version,
+            "has_changelog": self.has_changelog,
+            "schemas": self.schemas_metadata(),
+        }
+
+    def consolidation_tags(self, slug):
+        with open(os.path.join(self.static_dir, self.CONSOLIDATION_TAGS_FILENAME)) as f:
```

Comment:
No need for this PR, but I like to use Pathlib nowadays for this kind of stuff https://docs.python.org/3/library/pathlib.html

You can write `self.static_dir / self.CONSOLIDATION_TAGS_FILENAME`.

Reply:
Good to know! Didn't know it existed, it's way more fluent.